docs/source/virtual_ecosystem/theory/soil/environmental_links.md

```diff
@@ -65,18 +86,194 @@ which substrate breakdown is maximised, $\psi_{h}$ is the water potential at whi
+temperature in the model, and anticipate deprecating it in favour of more refined models
+in future.
+
+:::
+
+The thermal response of enzymatic rates and saturations is modelled using the Arrhenius
+equation. The form of this equation is as follows
+
+$$f(T) = \exp{\frac{-E_a}{R} * (\frac{1}{T} - \frac{1}{T_{\mathrm{ref}}})},$$
+
+where $E_a$ is the activation energy of the process of interest, $R$ is the molar gas
+constant, $T$ is the environmental temperature, and $T_{\mathrm{ref}}$ the reference
+temperature.
+
+### Impact of soil moisture on enzyme saturation
+
+The response of enzymatic rates to changing soil water potential is modelled using the
+same approach as for the below ground litter pools (described
+[here](#litter-decay-moisture-response)).
+
+## Direct environmental impacts on microbes
+
+Environmental factors also directly impact the growth of the different microbial groups
+in the model. The rate at which microbes can take up resources is affected by a wide
+range of environmental conditions. The efficiency microbes can grow is also affected by
+temperature, as is rate at which they lose biomass due to cell death and protein
+degradation.
+
+### Microbial uptake
+
+The uptake of resources by microbes is effected by a wide range of environmental
+factors, affecting both uptake rate and saturation. We use the same approach to
+calculate the environmental impacts on uptake rate and saturation as was used for
+enzymatic rate and saturation (described [here](#environmental-effects-on-enzymes)).
+
+### Microbial growth efficiency
+
+The efficiency of microbial growth is often expressed in carbon terms as a carbon use
+efficiency (CUE). This is defined as the proportion of carbon used for the synthesis of
+new biomass to the total amount of carbon taken up. This is an emergent property that
+arises from a large number of underlying processes (e.g. basal respiration, DNA
+synthesis efficiency, etc.), most of which would be expected to vary with temperature.
+Carbon use efficiency usually does not increase exponentially with temperature,
+therefore the Arrhenius model is rarely an appropriate model. Instead we use a simple
+logistic model to describe the temperature dependence of carbon use efficiency
+
+$$
+\mathrm{logit}\left(\epsilon\right) =
+\epsilon_{\mathrm{ref}} + \alpha * (T - T_{\mathrm{ref}}),
+$$
+
+where $\epsilon_{\mathrm{ref}}$ is the carbon use efficiency at the reference
+temperature, $\alpha$ is the change in carbon efficiency with temperature, $T$ is the
+environmental temperature and $T_{\mathrm{ref}}$ is the reference temperature. The logit
+link function is used to ensure that carbon use efficiency $\epsilon$ is bound between 0
+and 1 as it is a proportion.
+
+### Biomass loss
+
+The impact of temperature on the rate of biomass loss is assumed to follow the Arrhenius
+equation, which is described in detail
+[here](#impact-of-temperature-on-enzyme-rate-and-saturation).
+
+## Soil nutrient leaching rate
```

Comment:
I have changed a heading hierarchy above, but not sure if this one should be ## or ###